tests/validators/array.test.ts

```diff
@@ -111,4 +123,22 @@ describe('ArrayValidator', () => {
 			});
 		});
 	});
+
+	describe('Clone', () => {
+		// @ts-expect-error Accessing protected method
+		const clonePredicate = s.string.array.clone();
+
+		test.each([
+			[
+				['Hello', 'there'],
+				['Hello', 'there']
+			],
+			[
+				['Hello', 'there', 'foo'],
+				['Hello', 'there', 'foo']
+			]
+		])('GIVEN %p THEN returns given value', (value) => {
```

Comment:
We don't need to define 2 arguments.
```suggestion
		test.each([
			[['Hello', 'there']],
			[['Hello', 'there', 'foo']]
		])('GIVEN %p THEN returns given value', (value) => {
```